examples/02_meta-analyses/plot_cbma_subtraction_conjunction.py

```diff
@@ -0,0 +1,128 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; tab-width: 4; indent-tabs-mode: nil -*-
+# ex: set sts=4 ts=4 sw=4 et:
+"""
+
+.. _metas8:
+
+========================================================
+ Run coordinate-based subtraction and conjunction analyses
+========================================================
+
+The (coordinate-based) ALE subtraction method tests at which voxels 
+the meta-analytic results from two groups of studies differ reliably from 
+one another. [1]_:superscript:`,` [2]_
+"""
+import os
+
+from nilearn.image.image import math_img
+from nilearn.plotting.img_plotting import plot_stat_map
+
+from nimare.correct import FWECorrector
+from nimare.io import convert_sleuth_to_dataset
+from nimare.meta.cbma import ALE, ALESubtraction
+from nimare.tests.utils import get_test_data_path
+
+###############################################################################
+# Load Sleuth text files into Datasets
+# --------------------------------------------------
+# The data for this example are a subset of studies from a meta-analysis on
+# semantic cognition in children. [3]_ A first group of studies probed
+# childrens semantic world knowledge (e.g., correctly naming an object after
+# hearing its auditory description) while a second group of studies asked
+# children to decide if two (or more) words were semantically related to one
+# another or not.
+sleuth_file = os.path.join(get_test_data_path(), "semantic_knowledge_children.txt")
+sleuth_file2 = os.path.join(get_test_data_path(), "semantic_relatedness_children.txt")
+
+dset = convert_sleuth_to_dataset(sleuth_file)
+dset2 = convert_sleuth_to_dataset(sleuth_file2)
+
+###############################################################################
+# Individual group ALEs
+# --------------------------------------------------
+# Computing seperate ALE analyses for each group is not strictly necessary for
+# performing the subtraction analysis but will help to appreciate the
+# similarities and differences between the groups.
+ale = ALE(null_method="approximate")
+res = ale.fit(dset)
+res2 = ale.fit(dset2)
+
+corr = FWECorrector(method="montecarlo", voxel_thresh=0.001, n_iters=1000)
+cres = corr.transform(res)
+cres2 = corr.transform(res2)
+
+img = cres.get_map("z_level-cluster_corr-FWE_method-montecarlo")
+plot_stat_map(
+    img,
+    cut_coords=4,
+    display_mode="z",
+    title="Semantic knowledge",
+    threshold=2.326,  # cluster-level p < .01, one-tailed
+    cmap="RdBu_r",
+    vmax=4,
+)
+
+img2 = cres2.get_map("z_level-cluster_corr-FWE_method-montecarlo")
+plot_stat_map(
+    img2,
+    cut_coords=4,
+    display_mode="z",
+    title="Semantic relatedness",
+    threshold=2.326,  # cluster-level p < .01, one-tailed
+    cmap="RdBu_r",
+    vmax=4,
+)
+
+###############################################################################
+# Subtraction analysis
+# --------------------------------------------------
+sub = ALESubtraction(n_iters=1000, low_memory=False)
```

Comment:
We'll need to drop the number of iterations down a lot for this to run on GitHub/RTD. I think 100 would probably pass.

Reply:
Thanks a lot for your feedback! I've reduced `n_iters` and `n_cores` as suggested and the checks seem to pass now. Note, however, that I had to remove the thresholding for the subtraction image because `n_iters=100` didn't reliably detect anything at *p* < .01.